src/DynamoCoreWpf/Views/Preview/PreviewControl.xaml.cs

```diff
@@ -71,6 +74,18 @@ private struct Element
         private bool queuedRefresh;
         private MirrorData queuedMirrorData;
 
+        /// <summary>
+        ///     Indicates whether preview should stay open, when mouse leaves control.
+        /// </summary>
+        internal bool StaysOpen;
+
+        private const string baseUrl = @"pack://application:,,,/DynamoCoreWpf;component/UI/Images/";
+
+        private readonly BitmapImage pinIconBitmap =
```

Comment:
I recommend using Font Awesome icons here. And use a converter to determine the icon.